src/main/java/seedu/apollo/ui/Ui.java

```diff
@@ -55,12 +55,15 @@ public void printWelcomeMessage() {
      */
     public void printHelpMessage() {
```

Comment:
You need to change the expected help message output in expected.txt to match the updated command.